Source/WTF/wtf/text/StringHasher.h

```diff
@@ -22,6 +22,7 @@
 #pragma once
 
 #include <unicode/utypes.h>
+#include <wtf/FastMalloc.h>
```

Comment:
I know that we have this include in a lot of places in WTF. But aren't they all wrong, as `FastMalloc.h` should be coming from `config.h`?